src/System.Text.Encodings.Web/tests/UnicodeHelpersTests.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.Encodings.Web;
+using Xunit;
+
+namespace Microsoft.Framework.WebEncoders
+{
+    public unsafe class UnicodeHelpersTests
+    {
+        private const int UnicodeReplacementChar = '\uFFFD';
+
+        private static readonly UTF8Encoding _utf8EncodingThrowOnInvalidBytes = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
+
+        [Fact]
+        public void GetDefinedCharacterBitmap_ReturnsSingletonInstance()
+        {
+            // Act
+            uint[] retVal1 = UnicodeHelpers.GetDefinedCharacterBitmap();
+            uint[] retVal2 = UnicodeHelpers.GetDefinedCharacterBitmap();
+
+            // Assert
+            Assert.Same(retVal1, retVal2);
+        }
+
+        // TODO: fix xUnit escaping and then uncomment
+        //[Theory]
```

Comment:
I think you can use `[Theory(Skip = true)]` instead of commenting it out to ensure it keeps compiling, and I think it'll also show up in the test results as skipped.